modules/ROOT/pages/log-search-query-syntax.adoc

```diff
@@ -89,10 +90,10 @@ To escape these characters, use the `\` before the character. For example, to se
 
 == Search Fields
 
-When performing a search you can either specify a field name or use the default field. The field name and default field are specific to the search implementation.
+When performing a search, either specify a field name or use the default field. The field name and default field are specific to the search implementation.
 
-You can search any field by typing the field name followed by a colon ":" and then the term you are looking for.
-As an example, assume a log search query language index contains two fields--application and message--and message is the default field. If you want to find the result called "The Right Way" which contains the text "don't go this way", you can enter:
+Search any field by typing the field name, followed by a colon, then the term you are looking for.
+As an example, assume a log search query language index contains two fields&#8212;application and message&#8212;and message is the default field. If you want to find the result called "The Right Way", which contains the text "don't go this way", enter:
```

Comment:
For example, assume that...and that the message
Because we can directly assume things, I think "that" is necessary here.